Add documentation for tracking changes to Django and Wagtail (#208)
cfpb/development: standards/python.md

The isort configuration can go into a `tox.ini` file under an `[isort]` header.

## Dependency support and matrix testing

### Matrix testing

> We recommend using [tox](https://tox.readthedocs.io/en/latest/) for matrix testing against all the versions listed above, and we provide a [sample tox configuration](../tox.ini) that tests against both our current production the latest versions Python and Django.

Reviewer: This paragraph will need updating like so:

```suggestion
We recommend using [tox](https://tox.readthedocs.io/en/latest/) for matrix testing against our supported versions of our core dependencies, and we provide a [sample tox configuration](../tox.ini) that tests against both our current production the latest versions Python and Django.
```

And I would add this as a new paragraph below the next paragraph:

```markdown
See [Tracking Django and Wagtail](../guides/tracking-django-wagtail.md) for more detailed guidance.
```
